README.md

```diff
@@ -1,8 +1,60 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Amos Pivato . Jarvis Consulting
 
+My name is Amos Pivato, and I am a Toronto Metropolitan University Computer Science graduate with experience working in Data science and automation.  I then discovered the world of AI and machine learning and decided that I wanted to leverage my skills with these tools to help build solutions that could help improve the way we live our everyday lives.  Since then, I've worked in Python, Java, and VBA to build automated reporting and information consolidating tools, as well as developed applications from the ground up.  All of this practice has allowed me to grow my skills and proficiencies even further and become a confident and reliable back-end developper.
```

Comment:
"...developed applications from the ground up" seems a bit out of place. What sort of applications? Are the reporting and info tools not built from the ground up? Be more specific if possible.

Reply:
correct and ran through gramarly